src/components/shared/author.tsx

```diff
@@ -0,0 +1,27 @@
+// ----- Imports ----- //
+
+import { createElement as h } from 'react';
+
+import { renderText } from 'renderer';
+import { Pillar } from 'pillar';
+import { Option } from 'types/option';
+
+
+// ----- Component ----- //
+
+interface Props {
+    byline: Option<DocumentFragment>;
+    pillar: Pillar;
+}
+
+const Author = (props: Props): JSX.Element | null =>
+    props.byline.map<JSX.Element | null>((bylineHtml: DocumentFragment) =>
+        // This is not an iterator, ESLint is confused
+        // eslint-disable-next-line react/jsx-key
```

Comment:
Should we rename `map`? It will mean we can remove all the rule skips and might avoid some confusion for new developers (my first impression is that it's iterating)

Reply:
`fmap`?